spec/transaction_search_shared_examples.rb

```diff
@@ -11,8 +11,9 @@
       let(:params) { super().merge(account_owners: [3, 4]) }
 
       it "applies the parameter" do
-        expect(assigns(:order_details).where_values)
-          .to be_include("account_users.user_id in ('3','4')")
+        where_values = assigns(:order_details).where_values.reject { |v| v.is_a?(String) }
+        expect(where_values.map(&:to_sql))
+          .to include("`account_users`.`user_id` IN (3, 4)")
       end
```

Comment:
This should probably be tested against oracle before we merge it. I don't think it quotes tables/columns the same way. I'm not really sure why we tested this way. Is there a better test we could do?